code/modules/organs/organ.dm

```diff
@@ -238,3 +238,39 @@
 
 	if(cancerous_growth <= 0) //No cancerous growth this tick, no effects
 		return 1
+
+/datum/organ/duplicate(mob/living/carbon/human/destination)
+	if(!istype(destination))
+		return
+	var/datum/organ/dupe = new type()
+
+	for(var/x in vars)
+		if(!exclude.Find(x)) // Important!
+			if(x == "owner")
+				dupe.owner = destination
+				continue
+			if(istype(vars[x], /datum))
+				continue
+			if(istype(vars[x], /list))
+				var/list/OL = vars[x]
+				var/list/DL = dupe.vars[x]
+				if(!DL || !OL)
+					if(!DL)
+						DL = list()
+					if(!OL)
+						OL = list()
+				if(OL && DL)
```

Comment:
This line is unnecessary - you've ensured the condition is met above, and Byond is single threaded, so no other code can execute between the other code and this.